addons/sms/models/sms_template.py

```diff
@@ -112,6 +140,19 @@ def _get_ids_per_lang(self, res_ids):
 
         return tpl_to_rids
 
+    def _get_translated_bodies(self, res_ids):
```

Comment:
I think it is not a good idea to have yet another "lang / res_id" method ... wouldn't it be possible to extract code from sms_composer._prepare_body_values(), move it here and call it in both places ?